src/test/scala/uk/gov/hmrc/test/ui/pages/section6/ExpressConsignmentPage.scala

```diff
@@ -39,4 +46,8 @@ object ExpressConsignmentPage extends BasePage {
     selectYesOrNoRadio(values(yesNo))
     store(ExpressConsignment -> Detail(values(yesNo)))
   }
+
+  def gotoInlandOrBorderPage: Boolean =
+    (isStandard || !isSupplementary) && (isFixedTransport || isPostalOrMail)
```

Comment:
Uhm... the logic of `isStandard || !isSupplementary` is suspicious.

Are you sure? What's the condition here?

Reply:
Done